packages/docusaurus-theme-classic/codeTranslations/fa.json

```diff
@@ -55,13 +55,13 @@
   "theme.docs.sidebar.expandButtonTitle": "بزرگ کردن نوار کناری",
   "theme.docs.sidebar.responsiveCloseButtonLabel": "بستن منو",
   "theme.docs.sidebar.responsiveOpenButtonLabel": "باز کردن منو",
-  "theme.docs.versions.latestVersionLinkLabel": "latest version",
-  "theme.docs.versions.latestVersionSuggestionLabel": "For up-to-date documentation, see the {latestVersionLink} ({versionLabel}).",
-  "theme.docs.versions.unmaintainedVersionLabel": "This is documentation for {siteTitle} {versionLabel}, which is no longer actively maintained.",
-  "theme.docs.versions.unreleasedVersionLabel": "This is unreleased documentation for {siteTitle} {versionLabel} version.",
-  "theme.lastUpdated.atDate": "on {date}",
-  "theme.lastUpdated.byUser": "by {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "Last updated{atDate}{byUser}",
+  "theme.docs.versions.latestVersionLinkLabel": "آخرین نسخه",
+  "theme.docs.versions.latestVersionSuggestionLabel": "برای دیدن آخرین نسخه، {latestVersionLink} ({versionLabel}) را ببینید.",
+  "theme.docs.versions.unmaintainedVersionLabel": "نسخه {siteTitle} {versionLabel} دیگر بروزرسانی نمی شود.",
+  "theme.docs.versions.unreleasedVersionLabel": "نسخه {siteTitle} {versionLabel} هنوز به صورت کامل منتشر نشده است.",
+  "theme.lastUpdated.atDate": "در تاریخ {date}",
+  "theme.lastUpdated.byUser": "توسط {user}",
+  "theme.lastUpdated.lastUpdatedAtBy": "آخرین به روزرسانی {atDate} {byUser}",
```

Comment:
thanks, will just update this part as the spaces must be included inside the atDate and byUser as both parts can be disabled independently.

Let me know if my update does not look good to you

![image](https://user-images.githubusercontent.com/749374/109541351-028e7000-7ac4-11eb-86b4-49acc1e20440.png)


https://github.com/facebook/docusaurus/pull/4312#discussion_r584947278

Reply:
@slorber 
As of now, it looks fine to me, so you may proceed please.